dungeon-crawler/android/src/main/java/com/novoda/dungeoncrawler/ArduinoLoop.java

```diff
@@ -17,6 +16,7 @@
      * @param loopable the method to continuously lopp
      */
     void start(Loopable loopable) {
+        thread = new HandlerThread("Arudino Loop Mimic");
```

Comment:
if you call start twice now, you'll leave a thread in the background thats never stopped?

Reply:
will call stop before starting 👍